lab_tests/parse_daily_tests.py

```diff
@@ -3,239 +3,258 @@
-        "tests.lab.golnik.performed.todate",
         "tests.lab.golnik.positive",
-        "tests.lab.golnik.positive.todate",
         "tests.lab.nlzohkp.performed",
-        "tests.lab.nlzohkp.performed.todate",
         "tests.lab.nlzohkp.positive",
-        "tests.lab.nlzohkp.positive.todate",
         "tests.lab.nlzohnm.performed",
-        "tests.lab.nlzohnm.performed.todate",
         "tests.lab.nlzohnm.positive",
-        "tests.lab.nlzohnm.positive.todate",
         "tests.lab.sbce.performed",
-        "tests.lab.sbce.performed.todate",
         "tests.lab.sbce.positive",
-        "tests.lab.sbce.positive.todate",
         "tests.lab.nlzohkr.performed",
-        "tests.lab.nlzohkr.performed.todate",
         "tests.lab.nlzohkr.positive",
-        "tests.lab.nlzohkr.positive.todate",
         "tests.lab.sbng.performed",
-        "tests.lab.sbng.performed.todate",
         "tests.lab.sbng.positive",
-        "tests.lab.sbng.positive.todate",
         "tests.lab.sbsg.performed",
-        "tests.lab.sbsg.performed.todate",
         "tests.lab.sbsg.positive",
-        "tests.lab.sbsg.positive.todate",
         "tests.lab.nlzohms.performed",
-        "tests.lab.nlzohms.performed.todate",
         "tests.lab.nlzohms.positive",
-        "tests.lab.nlzohms.positive.todate",
+        "tests.regular.performed",
+        "tests.regular.positive",
+        "tests.ns-apr20.performed",
+        "tests.ns-apr20.positive",
     ]
-]
 
-xy.to_csv(path_or_buf=OUTPUT / "tests.csv", sep=",", index=False)
+    custom_column_indices = list(range(30))
+    custom_column_indices.pop(27)
+    custom_column_indices.pop(26)
+    custom_column_indices.pop(0)
+
+    xy = pd.read_excel(
+        xlsx,
+        header=0,
+        sheet_name="podatki",
+        skiprows=[0, 1, 2, 3],
+        parse_dates=True,
+        names=custom_column_names,
+        usecols=custom_column_indices,
+        converters=dict((i, int) for i in range(len(custom_column_indices)) if i > 0),
+        skipfooter=1,  # skips the SKUPAJ row at the end
+    )
+
+    # Some dates are invalid and we need to prepare them before parsing.
+    for index, value in xy.iterrows():
+        interval = xy.loc[index, "datum"]
+
+        if not isinstance(interval, datetime.datetime):
+            # Catch any datum with a star at the end.
+            interval = re.sub(pattern=r"\*$", repl="", string=interval)
+
+            try:
+                end_datum = datetime.datetime.strptime(interval, "%d.%m.%Y")
+            except ValueError as te:
+                pass
+
+            try:
+                end_datum = interval.split("-")[1]
+                end_datum = datetime.datetime.strptime(end_datum, "%d.%m.%Y")
+            except IndexError as ie:
+                pass
+            xy.loc[index, "datum"] = end_datum
+
+    # The table comes with dates in the future. Since some cells may be polluted,
+    # the easiest thing to do is to remove them.
+    xy = xy[xy["datum"] <= datetime.datetime.now()]
```

Comment:
this should be `now - 1`

but really should be the parameter that is used to find the filename